composer.json

```diff
@@ -19,12 +19,5 @@
   "minimum-stability": "dev",
   "config": {
     "preferred-install": "dist"
-  },
-  "extra": {
-    "heroku": {
-      "framework": "cakephp2",
-      "document-root": "webroot",
-      "index-document": "index.php"
-    }
```

Comment:
Everything appears to be working fine on a test I'm running with Cake v2.6.0. That appears to be specific to the CHH buildpack, and is instead now defined in a Procfile.